core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/distinct.kt

```diff
@@ -23,45 +29,60 @@ import kotlin.reflect.KProperty
 // region DataFrame
 
 /**
- * ## The Distinct Operation
+ * {@get [DESCRIPTION] Removes duplicated rows based on {@get [PHRASE_ENDING]}}.
```

Comment:
interesting, I've never used a `{@get}` inside the "default" section of another `{@get}` XD but it works :). Did you know you can also write `$[PRASE_ENDING]` btw? just an alternative notation, does the same.

Reply:
you can also provide a "default" with that notation, but it needs to be small, as spaces split it up already, like `$key=default` :)